vimrc

```diff
@@ -227,7 +227,21 @@ map <silent> <LocalLeader>nr :NERDTree<CR>
 map <silent> <LocalLeader>nf :NERDTreeFind<CR>
 
 " FZF
-map <silent> <leader>ff :Files<CR>
+function! s:in_git_repo()
+  let root = split(system('git rev-parse --show-toplevel'), '\n')
+  if len(root) == 0 || v:shell_error
+    return 0
+  else
+    return 1
+  end
+endfunction
+
+if s:in_git_repo()
```

Comment:
this will only run at start up and not when invoked right?  I think that might produce some strange results if you navigate around directories in vim.  How do you feel about pushing the conditional into a function:

```

function! RunFuzzyFind()
  if s:in_git_repo()
    :Gfiles ...
  else
    :Files
  end
endfunction

map <leader>ff :call RunFuzzyFind()<cr>
```

Reply:
@allcentury I actually thought about that and am unsure what the more correct behavior is.

I think we _really_ only want it run once when vim is started in the repo it is started in. Everytime I am in fuzzy find I don't want to be recalculating the fuzzy find. I.e. if I open a random file from another repo, should fzf in that pane change the directory I am doing fzf in?

The current fzf behavior does not change based on navigating directories in vim, so I think the existing PR would continue existing behavior.